index.html

```diff
@@ -0,0 +1,65 @@
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header>
+            <p class="time"> 9:14 <img class="net" src="C:\Users\Harrison\phone network image.jpg" alt="phone network"></p>
+            <img class="logo" src="C:\Users\Harrison\Downloads\My Logo.png" alt="logo">
+            <img class="navicon" src="https://assets.stickpng.com/images/588a64c6d06f6719692a2d0c.png" alt="nav icon"><br>
+         
+          </header>
+        </div>
+        <div class="item2">
+          <div class="design">
+            <img class="ellipse11" src="file:///C:/Users/Harrison/Downloads/Ellipse%2011%20(1).svg" alt="ellipse 11">
+            <img class="rectangle4" src="C:\Users\Harrison\Downloads\Rectangle 4.svg" alt="rectangle4">
+            <img class="ellipse1" src="file:///C:/Users/Harrison/Downloads/Ellipse%203.svg" alt="ellipse1">
+            <img class="ellipse5" src="C:\Users\Harrison\Downloads\Ellipse 5.svg" alt="ellipse 5">
+            <img class="rectangle7" src="C:\Users\Harrison\Downloads\Rectangle 7.svg" alt="rectangle7">
+            <img class="ellipse8" src="C:\Users\Harrison\Downloads\Ellipse 8.svg" alt="ellipse8">
+            <img class="ellipse13" src="C:\Users\Harrison\Downloads\Ellipse 13.svg" alt="ellipse13">
+            <img class="ellipse12" src="C:\Users\Harrison\Downloads\Ellipse 12.svg" alt="ellipse12">
+            </div>
+
+            <p class="mariane">I'm Mariane<br>
+               Glad to see<br>
+              you!<br>
+            </p>
+          <section>
+            <p class="text">
+              I’m a software developer! I can help you<br>
+               build a product , feature or website Look <br>
+               through some of my work and<br>
+                experience! If you like what you see and <br>
+                have a project you need coded, don’t<br>
+                 hestiate to contact me.
+            </p>
+          </section>
+          <p class="cap">LET’S CONNECT</p>
+<div class="img">
+  <img  src="file:///C:/Users/Harrison/Downloads/Vector.svg" alt="twitter">
+          <img  class="i" src="file:///C:/Users/Harrison/Downloads/Vector%20(1).svg" alt="i">
+          <img src="file:///C:/Users/Harrison/Downloads/Vector%20(2).svg" alt="n">
+          <img class="m" src="file:///C:/Users/Harrison/Downloads/Vector%20(3).svg" alt="M">
+          <img  class="cat" src="file:///C:/Users/Harrison/Downloads/Vector%20(4).svg" alt="cat">
+          <img  class="twofingers" src="file:///C:/Users/Harrison/Downloads/Vector%20(5).svg" alt="two fingers">
+</div>
+<div class="secdesign">
+<img class="ellipse4" src="C:\Users\Harrison\Downloads\Ellipse 4.svg" alt="ellipse4">
+<img class="ellipse15" src="C:\Users\Harrison\Downloads\Ellipse 15.svg" alt="ellipse15">
+<img class="ellipse14" ellipse14" src="C:\Users\Harrison\Downloads\Ellipse 14.svg" alt="ellipse14">
```

Comment:
- [ ] I think you can remove this part with the part I mentioned above and a `background-image` in CSS not an `img` tags in HTML. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info